src/_nebari/stages/infrastructure/template/aws/modules/kubernetes/main.tf

```diff
@@ -69,9 +68,36 @@ data "aws_eks_cluster_auth" "main" {
 
 resource "aws_eks_addon" "aws-ebs-csi-driver" {
   # required for Kubernetes v1.23+ on AWS
-  addon_name        = "aws-ebs-csi-driver"
-  cluster_name      = aws_eks_cluster.main.name
-  resolve_conflicts = "OVERWRITE"
+  addon_name                  = "aws-ebs-csi-driver"
+  cluster_name                = aws_eks_cluster.main.name
+  resolve_conflicts_on_create = "OVERWRITE"
+  resolve_conflicts_on_update = "OVERWRITE"
+
+  configuration_values = jsonencode({
+    controller = {
+      nodeSelector = {
+        "eks.amazonaws.com/nodegroup" = "general"
```

Comment:
I'd like to pass this is as a variable rather than hard code it (as in src/_nebari/stages/kubernetes_services/template/modules/kubernetes/services/jupyterhub/main.tf)

Reply:
I actually don't think we support renaming node pools so I'm fine adding this as is.